mediasms.go

```diff
@@ -0,0 +1,109 @@
+package mediasms
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/comvex-jp/mediasms-go/errors"
+	"github.com/comvex-jp/mediasms-go/models"
+	"github.com/comvex-jp/mediasms-go/translations"
+)
+
+// Client struct
+type Client struct {
+	Username string
+	Password string
+	Prefix   string
+}
+
+// resultCode from media4u
+type resultCode struct {
+	Result string `json:"result"`
+}
+
+func createSMSID(prefix, messageID string) string {
+	return prefix + messageID
+}
+
+const smsurl = "https://www.sms-console.jp/"
+
+// Send request to media4u
+func (c Client) Send(messageID string, val models.SendRequest) (models.APIResponse, error) {
+	smsID := createSMSID(c.Prefix, messageID)
+
+	val.Smsid = smsID
+
+	jsonValue, _ := json.Marshal(val)
+	client := &http.Client{}
+	req, err := http.NewRequest("POST", smsurl+"api/", bytes.NewBuffer(jsonValue))
+	req.SetBasicAuth(c.Username, c.Password)
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer resp.Body.Close()
+
+	body, _ := ioutil.ReadAll(resp.Body)
+	var sendResult resultCode
+	json.Unmarshal(body, &sendResult)
+
+	var res = errors.SendResultsMapper[sendResult.Result]
+
+	results := models.APIResponse{
+		StatusCode:  sendResult.Result,
+		Name:        res["name"],
+		Description: res["description"],
+	}
+
+	return results, err
+}
+
+// GetStatus of a sent sms
+func (c Client) GetStatus(messageID string) (models.APIResponse, error) {
+	smsID := createSMSID(c.Prefix, messageID)
+
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", smsurl+"api5/?smsid="+smsID, nil)
+	req.SetBasicAuth(c.Username, c.Password)
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
empty line

Reply:
Same here. Empty line below 84, between `body` and `s`?